js/entry-points/tracing.js

```diff
@@ -661,14 +673,17 @@ audion.entryPoints.tracing = function() {
 
   /**
    * Wraps the web audio connect method.
-   * @param {function(...*):*} nativeConnect The native connect method.
+   * @param {function(...*):*} nativeBoundConnect The native connect method.
+   *     This is the apply method already bound to the connect method. We bind
+   *     the apply method because other libraries may actually override the
+   *     apply method.
```

Comment:
I personally do not want to support any library that modifies built-in prototype chain, but refactoring Box2D demo needs too much effort to drop the "prototype" library. So here we are.